code/modules/urist/modules/wfc/modules/deepmaint/teleporters.dm

```diff
@@ -0,0 +1,143 @@
+
+/proc/deepmaint_yeet_trg(var/atom/movable/trg, var/xoffset = 1, var/yoffset = 1)
+	if(!istype(trg, /atom/movable))
+		return FALSE
+
+	if(isnull(trg.loc))
+		// No un-qdel-ing by accident!
+		return
```

Comment:
Eh, maybe, but this would be more painful to port out of SS13 for standalone testing and probably catch less cases.

Reply:
Are there cases where a datum might be marked for GC but still have a loc, however? That could be missed here if so

Reply:
Shouldn't be a problem in that case; even if we teleport it, it will be moved into nullspace by the GC, just from a different location. 

The primary purpose of this is to avoid accidentally creating new refs by moving things out of nullspace, whether it's there due to GC or some nonsense (e.g. being inside an object which then gets deleted, but the logic doesn't transitively delete the contents fsr).